docs: Update docs to switch ttlSecondsAfterEmpty to consolidation (#3964)
aws/karpenter: website/content/en/preview/getting-started/getting-started-with-karpenter/_index.md

We applied the tag `karpenter.sh/discovery` in the `eksctl` command above.
Depending how these resources are shared between clusters, you may need to use different tagging schemes.

> The `consolidation` value configures Karpenter to consolidate pods on into nodes by moving pods into free capacity in the cluster to then delete nodes or replacing nodes for cheaper nodes. This behavior can be disabled by leaving the value undefined. Read [Consolidation](../../concepts/deprovisioning/#consolidation) for more.

Reviewer: ```suggestion
The `consolidation` value configures Karpenter to reduce cluster cost by removing and replacing nodes to increase node resource utilization and decrease cluster cost. As a result, consolidation will terminate any empty nodes on the cluster. This behavior can be disabled by leaving the value undefined or setting `enabled` to `false`. Review the [provisioner CRD]({{<ref "../../concepts/provisioners" >}}) for more information.
```

There's a few grammar things in here. Edited some things to increase clarity.

> The `consolidation` value configures Karpenter to reduce cluster cost by removing and replacing nodes to increase node resource utilization and decrease cluster cost. As a result, consolidation will terminate any empty nodes on the cluster. This behavior can be disabled by leaving the value undefined or setting `enabled` to `false`. Review the [provisioner CRD]({{<ref "../../concepts/provisioners" >}}) for more information.

Reviewer: ```suggestion
The `consolidation` value configures Karpenter to reduce cluster cost by removing and replacing nodes. As a result, consolidation will terminate any empty nodes on the cluster. This behavior can be disabled by leaving the value undefined or setting `consolidation.enabled` to `false`. Review the [provisioner CRD]({{<ref "../../concepts/provisioners" >}}) for more information.
```

### Scale down deployment

> Now, delete the deployment. `consolidation`, Karpenter should terminate the now empty nodes.

Reviewer: ```suggestion
Now, delete the deployment. After a short amount of time, Karpenter should terminate the now empty nodes due to consolidation.
```

> Now, delete the deployment. After a short amount of time, Karpenter should terminate the now empty nodes due to consolidation.

Reviewer: ```suggestion
Now, delete the deployment. After a short amount of time, Karpenter should terminate the empty nodes due to consolidation.
```
